__test__/components/navbar.spec.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
+import { mount } from 'enzyme';
```

Comment:
Hi @tolumide-ng and @olorunwalawrence,

Please remove the `mount` function since it has been configured globally on the project.

Reply:
sorry this mount is on navbar component not on the signup